doc/misc/specs.7.rst

```diff
@@ -132,6 +132,19 @@ against NEVRAs and file provides.
 ``<package-name-spec>`` is matched against NEVRAs only.
 
 
+Dependency expressions
+======================
+
+When passing a ``<package-spec>``, the user can provide additional restriction
```

Comment:
Since the paragraph extends <package-spec>, I would keep the paragraph in Packages section, just after first paragraph "Many  commands…file provides.